src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -996,6 +1002,69 @@ public void ctcp(ClientReceiveCommandEvent event) {
         }
     }
 
+    private void handleDCCEvent(User user, List<ServerMessage> originalMessages, List<String> parameters) {
+        // TODO should unknown requests or improperly formatted requests get logged? just to clarify to the user that they are still noticed
```

Comment:
Not really user but client originating. Still could be maliciously crafted easily.